# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../atop/cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- atop-1.20/Makefile.vanilla	2007-07-10 03:51:06.843344714 +0200
+++ atop-1.20/Makefile	2007-07-10 03:51:29.288623797 +0200
@@ -24,7 +24,7 @@
 all: 		atop
 
 atop:		atop.o    $(ALLMODS) Makefile
-		cc atop.o $(ALLMODS) -o atop $(LDFLAGS)
+		$(CC) atop.o $(ALLMODS) -o atop $(LDFLAGS)
 
 clean:
 		rm -f *.o
